raiden/transfer/channel.py

```diff
@@ -128,6 +132,19 @@ def is_lock_locked(
     return secrethash in end_state.secrethashes_to_lockedlocks
 
 
+def is_lock_expired(
+        locked_lock: LockedTransferSignedState,
```

Comment:
From the way you use it it seems that the type should be: `typing.Optiona[LockedTransferSignedState]`